doc/mgr/administrator.rst

```diff
@@ -90,6 +90,13 @@ standby ceph-mgr daemons as well as the active daemon.  This enables
 modules that provide services to redirect their clients to the active
 daemon, if the client tries to connect to a standby.
 
+.. note::
+
+   Enabling a module will asynchronously trigger the respawn of the ceph-mgr
+   daemon. The respawned daemon starts to load all the enabled modules. Hence
+   the loading of a enabled module is not immediate. You may have to retry the
+   the first module command until the module loads.
```

Comment:
I'd avoid the "first module command" wording since this can happen in the middle of operation if some other module gets enabled or on failover, right?  Something like:

> Enabling a module will asynchronously trigger the respawn of the ceph-mgr
> daemon. The respawned daemon then loads enabled modules one by one. Hence
> the loading of a (newly enabled) module is not immediate; any command issued
> against the module before it is finished loading can fail with a transient
> `ENOTSUP` error.

Reply:
Also, I'd move this one paragraph up, before the introduction of the standby mode, since the instructions on enabling and disabling seem to end there.

Reply:
Thanks! Done.